docs/conf.py

```diff
@@ -1,7 +1,8 @@
 import os
 import sys
 
-sys.path.insert(0, "..")
+sys.path.insert(0, ".")
+sys.path.insert(1, "..")
```

Comment:
Why is this added?

Reply:
The doc build won't work otherwise. It needs curdir and parent dir

Reply:
It works without that line if run in the build directory, but not in the parent dir